drivers/gpu/drm/vc4/vc4_firmware_kms.c

```diff
@@ -1370,16 +1516,28 @@ vc4_fkms_connector_init(struct drm_device *dev, struct drm_encoder *encoder,
 	fkms_connector = devm_kzalloc(dev->dev, sizeof(*fkms_connector),
 				      GFP_KERNEL);
 	if (!fkms_connector) {
-		ret = -ENOMEM;
-		goto fail;
+		return ERR_PTR(-ENOMEM);
+	}
```

Comment:
Unnecessary braces, although checkpatch seems unconcerned.

Reply:
Checkpatch not being fussy for once. I'm glad I was sitting down!